### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by 0xLeif** [oneleif/olDocs#15] [General]
# How is your day?

**Comment by haIIux** [comment]
Busy! Big change with some life events but, making the best of them. This weeks goals are almost met as, the big plan was to get my personal project connected with its new backend and it's almost there locally for testing. 

How about you?

**Reply by 0xLeif** [reply]
I'm doing great! Excited to work on projects this weekend. Going to work some more on Fork!

**Comment by krishnakishor13** [comment]
Tired! Heavy rains here in Bengaluru, India. Literally hate to say it, I love rain

**Reply by haIIux** [reply]
I would love a few days of rain for the next few days, my lawn and well could use it!

**Reply by krishnakishor13** [reply]
Yep!

**Reply by aayushchugh** [reply]
There are floods in india you can take some water from here
